generators/app/templates/services/passport/index.js

```diff
@@ -31,6 +34,13 @@ export const facebook = () =>
   passport.authenticate('facebook', { session: false })
 
 <%_ } _%>
+
```

Comment:
It will create more than 1 blank line which isn't allowed by the linter. Just remove this line.